CHANGES.md

```diff
@@ -24,6 +24,12 @@ To be released.
     and its all implementations.  [[#189]]
  -  The signature of `IStore.PutBlock<T>(Block<T>)` method was changed to
     `PutBlock<T>(Block<T>, Address)`.  [[#189]]
+ -  Improved the read throughput of `BlockChain<T>` when calling
+    `BlockChain<T>.Append()`.
+ -  Fixed a bug that `Swarm` attempted to use TURN relay even though `_host` was
```

Comment:
As the `_host` field is just internal details that library users are not aware of, we'd better to refer to the `Swarm` contstructor's `host` parameter instead:

```suggestion
 -  Fixed a bug that `Swarm` had attempted to use TURN relay even if the `host` argument was
```